app/bundle/src/main/scala/org/bitcoins/bundle/gui/BitcoindConfigPane.scala

```diff
@@ -53,7 +53,9 @@ class BitcoindConfigPane(
       minWidth = 300
     }
 
-  private val torCheckBox: CheckBox = new CheckBox()
+  private val torCheckBox: CheckBox = new CheckBox() {
+    selected = appConfig.config.getBoolean("bitcoin-s.proxy.enabled")
```

Comment:
would be better to do `appConfig.nodeConf.socks5ProxyParams.isDefined` in case we change how this is defined, then we don't need to remember to change it here as well

Reply:
thanks, changed these